### PR TITLE
Update to permissions requested by APP

### PR DESCRIPTION
**Pull request by anwara000**
This is the screenshot that needs to be updated:

image

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/information-barriers-policies.md [hunk 1]
```diff
@@ -83,7 +83,9 @@ In addition to the [required licenses and permissions](information-barriers.md#r
 
    2. When prompted, sign in using your work or school account for Office 365.
 
-   3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**.
+   3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions reqested by the App is given below <add a screenshot>
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md:86]:
```suggestion
   3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions requested by the App are given below. 
  add a screenshot>
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md:86]:
```suggestion
   3. In the **Permissions requested** dialog box, review the information, and then choose **Accept**. The permissions requested by the app are given below.
   <add a screenshot>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@anwara-ms

**Pull request merged**